app/elements/mdl-footer.html

```diff
@@ -1,201 +1,208 @@
 <dom-module id='mdl-footer'>
   <template>
     <style>
-    :host {
-      position: relative;
-    }
-    
-    @media (min-width: 600px) {
       :host {
-        position: absolute;
-        left: 0;
+        position: relative;
+      }
+
+      @media (min-width: 600px) {
+        :host {
+          position: absolute;
+          left: 0;
+          width: 100%;
+          bottom: 0;
+        }
+      }
+
+      img {
+        list-style: none;
```

Comment:
Oh, I've just re-indented styles in this file, looks like some old style - probably I can delete this one.